Formula/h/helix.rb

```diff
@@ -17,6 +17,7 @@ class Helix < Formula
 
   depends_on "rust" => :build
 
+  conflicts_with "evil-helix", because: "both install `hx` binaries"
```

Comment:
all these conflicts can be done as followup syntax pr so that we dont have to rebottle for the formulae.

Reply:
Not familiar with rebottling - do you need me to submit without the conflict? I added them because of warnings of brew audit...

Reply:
Removed these two for now (as I assume these are causing the CI fail)